src/components/molecules/UiLoader/_internal/UiLoaderTransition.vue

```diff
@@ -8,6 +8,7 @@
   <div
     ref="contentEl"
     :style="contentStyle"
+    style="display: contents"
```

Comment:
I don't think we should ever use non-dynamic `style` property. 👀

Reply:
In this case, we want to use it.

Reply:
this div is our helper; its wrap content to use `<transition>` always

Reply:
👍